app/services/idv/steps/inherited_proofing/verify_wait_step_show.rb

```diff
@@ -0,0 +1,34 @@
+module Idv
+  module Steps
+    module InheritedProofing
+      class VerifyWaitStepShow < VerifyBaseStep
+        include UserPiiManagable
+        delegate :controller, :idv_session, to: :@flow
+
+        STEP_INDICATOR_STEP = :getting_started
+
+        def call
+          poll_with_meta_refresh(IdentityConfig.store.poll_rate_for_verify_in_seconds)
+
+          process_async_state(async_state)
+        end
+
+        private
+
+        def process_async_state(current_async_state)
+          if current_async_state.none?
+            inherited_proofing_save_user_pii_to_session!
+            inherited_proofing_form_response
+          elsif current_async_state.in_progress?
+            nil
+          elsif current_async_state.missing?
+            flash[:error] = I18n.t('idv.failure.timeout')
+            # Need to add path to error pages once they exist
```

Comment:
Perhaps a more instructive comment:
# LG-<add error handling/pages card number>
# LG-<add analytics card number>
# This method overrides VerifyBaseStep#process_async_state:
# See the VerifyBaseStep#process_async_state "elsif current_async_state.missing?"
# logic as to what is typically needed/performed when hitting this logic path.

Reply:
I can do that 👍